### ISSUE TITLE
Upload to JSR?

### ISSUE THREAD

**Issue by Kayxue** [ai/nanoid#502]
Will this module be uploaded to jsr?

**Comment by ai** [comment]
I do not plan to do it because it is extra work for me. 

Is there any rational reason why it is important? Do we have users who can't use npm?

**Comment by AugustinMauroy** [comment]
JSR is open source and becoming open governance. So big lib should starting publishing on both so that one day we can say goodbye to an old npm.

if you need help I can

**Comment by ai** [comment]
Yes, the first version was published and a little later I will add CI, docs, and some fixes

**Comment by ai** [comment]
Done https://jsr.io/@sitnik/nanoid
